nekoyume/Assets/_Scripts/UI/Module/Button/ConditionalCostButton.cs

```diff
@@ -192,7 +191,7 @@ protected override void OnClickButton()
         {
             base.OnClickButton();
 
-            if (!showCostAlert && CurrentState.Value != State.Conditional)
+            if (!showCostAlert && CurrentState.Value == State.Conditional)
```

Comment:
```suggestion
            if (!showCostAlert)
```
Actually cost alert had been showing both conditional and normal state.
It is already decided whether cost alert is shown via `showCostAlert`.
So I think this comparison is unnecessary.